src/sqlite/index.ts

```
@@ -350,14 +437,31 @@ api.dbFileName = async () => {
 api.subscribeToAllChangesInTable = subscribeToAllChangesInTable
 api.unsubscribeFromChange = unsubscribeFromChange
-
-export type Api = Comlink.RemoteObject<typeof api>
-
-Comlink.expose(api, self as any)
+api.subscribeToChangeLog = subscribeToChangeLog
+api.downloadLocalDB = downloadLocalDB
+api.disconnect = () => {}
+
+let ports: MessagePort[] = []
+onconnect = e => {
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)